src/renderer/components/cluster-view/ClusterView.tsx

```diff
@@ -285,12 +285,11 @@ const ClusterView: FC = () => {
             >
                 {cluster?.chips.map((clusterChip) => {
                     let graphOnChip: GraphOnChip | undefined;
-                    let graphName: string | undefined;
+
                     selectedEpoch.forEach((graph) => {
                         const chipByGraphName = getGraphOnChip(graph.temporalEpoch, graph.chipId);
```

Comment:
`chipByGraphName`  is a poor name choice imo. `graphOnChip` should be present in the name but im struggling to finish it. graphOnChipById? :/

Reply:
or possibly we could use `graphOnChip` and rename local graphOnChip to `currentGraphOnChip`?